src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/summaryjavadoc/InputSummaryJavadocInlineForbidden.java

```diff
@@ -114,8 +114,7 @@ void foo12() {
      * Sentence starts as a plain text sentence
      * {@summary ... but ends in the summary tag}
      */
-    // violation 2 lines above 'Summary .* missing an ending period.'
-    // until https://github.com/checkstyle/checkstyle/issues/11213
+    // violation 2 lines above 'Summary of Javadoc is missing an ending period.'
```

Comment:
When the first sentence doesn't end with a period,`@summary` is also included.
eg-
```
/**
 * Sentence starts as a plain text sentence
 *
 *
 * 
 * {@summary ... but ends in the summary tag.}
 */
public class InputSummaryJavadocIncorrect3 {
}
``` 
![Screenshot from 2022-03-10 13-51-02](https://user-images.githubusercontent.com/73882557/157618778-bba4b26a-d33c-43b3-9759-f0502adcf5ee.png)

Reply:
I think it is big in javadoc tool. It is ok to consider summary to be text before inlined summary tag

Reply:
I feel it is better to align ourselves with the current behavior of the Javadoc tool, if it is truly a bug then we should either report it or search for the bug if it is already present and add an until comment.

Reply:
@Vyom-Yadav , can you report defect on jdk (javadoc tool) ?
It is really weird that location of `.` make inline tag work differently.

meanwhile I would vote to violate this example and demand dot before inline tag.

Reply:
I have filed a report in Oracle JDK, I wasn't able to file a bug report in OpenJDK as I don't have sufficient permissions to do so, oracle doesn't make the issue publically visible, they have given me an internal reference ID and will contact me on mail for further steps or inquiry.